docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
@cgwalters We don't seem to install /etc/sysconfig/docker-storage by default. So there is nothing to restore over reset.